ios/RNZeroconf/RNNetServiceSerializer.m

```diff
@@ -14,7 +14,7 @@
 const NSString *kRNServiceKeysAddresses = @"addresses";
 const NSString *kRNServiceKeysHost = @"host";
 const NSString *kRNServiceKeysPort = @"port";
-const NSString *kRNServiceTxtRecord = @"txt";
+const NSString *kRNServiceTxtRecords = @"txtRecords";
```

Comment:
We'll try to make the api between Android and iOS compatible, so you should change it to `txt` back, even if it's now a record